src/portable/create.rs

```diff
@@ -30,6 +30,11 @@ pub fn create(options: &Create) -> anyhow::Result<()> {
         );
         return Err(ExitCode::new(exit_codes::DOCKER_CONTAINER))?;
     }
+    if options.start_conf.is_some() {
+        print::warn("The option `--start-conf` is deprecated. \
+                     Use `edgedb instance start/stop` to control \
+                     the instance.");
```

Comment:
I think after merging this a new issue should be created in https://github.com/edgedb/setup-edgedb that uses this option right now

Reply:
Created this pre-emptively so we don't forget: https://github.com/edgedb/setup-edgedb/issues/32